cmd/agent/main.go

```diff
@@ -106,11 +111,27 @@ type measuresServer struct {
 }
 
 func (s *measuresServer) saveCounter(m Measure, v int64) {
-	s.R().Post("/counter/" + m.name() + "/" + fmt.Sprintf("%d", v))
+	j := &model.Metrics{}
+	j.ID = m.name()
+	j.MType = "counter"
+	j.Delta = &v
```

Comment:
```suggestion
	j := &model.Metrics{
	   ID: m.name()
	   // ...
	}

```

здесь и ниже - лучше сразу заполнять стурктуру